rust_src/src/buffers.rs

```diff
@@ -690,7 +691,7 @@ impl LispObject {
     }
 
     pub fn as_live_buffer(self) -> Option<LispBufferRef> {
-        self.as_buffer().and_then(|b| b.as_live())
```

Comment:
Is this new? I've always hated having to create a closure just to call a method with no arguments.